pkg/snapshot/snapshotter/garbagecollector.go

```diff
@@ -68,8 +70,8 @@ func (ssr *Snapshotter) RunGarbageCollector(stopCh <-chan struct{}) {
 				ssr.logger.Infof("GC: Total number garbage collected chunks: %d", chunksDeleted)
 			}
 
-			snapStreamIndexList := getSnapStreamIndexList(snapList)
-
+			fullSnapshotIndexList := getFullSnapshotIndexList(snapList)
+			// snapStream indicates the list of snapshot, where first snapshot is base/full snapshot followed by list of incremental snapshots based on it.
```

Comment:
```suggestion
			// snapStream indicates a list of snapshots, where the first snapshot is base/full snapshot followed by a list of incremental snapshots based on it. Garbage collection is performed on one snapStream at a time.
```
Agree with @anveshreddy18's [review comment](https://github.com/gardener/etcd-backup-restore/pull/793/files#r1805250327), but I don't think the code comment is currently very useful in its state.

I don't particularly like that currently there is explanation about what `snapStream` is, but there is no `snapStream` token in the function where the explanation is present.
There is no variable/type in the function with `snapStream` in it. Unless someone searches for that word in the rest of the file, the comment causes more confusion.

See the suggestion in https://github.com/gardener/etcd-backup-restore/pull/793#discussion_r1818848135, would make the above suggested code read better.